run.go

```diff
@@ -53,6 +54,7 @@ func complete(opts ...RunOptions) (result RunOptions, _ error) {
 	for _, opt := range opts {
 		result.TrustedRepoPrefixes = append(result.TrustedRepoPrefixes, opt.TrustedRepoPrefixes...)
 		result.DisableCache = first(opt.DisableCache, result.DisableCache)
+		result.CredentialOverrides = append(result.CredentialOverrides, opt.CredentialOverrides...)
```

Comment:
This is singular in `gptscript` and plural in the SDKs, but it should probably be singular everywhere. Will probably follow up on that shortly.

Reply:
It's singular in `gptscript` because of the convention of being about to pass `--credential-override` multiple times. That convention doesn't apply to the SDKs.

Reply:
Yup, but IMO the benefits of being consistently singular everywhere outweigh the aesthetics of pluralizing the field in the sdk. From a usage perspective, having the sdk option match the gptscript feels like it leaves less room for error.

In any case, I only brought this up because I anticipated pushback on the plural names. If it's not an issue for anyone, then I'm happy to leave it be.